zerver/lib/streams.py

```diff
@@ -785,7 +786,7 @@ def get_occupied_streams(realm: Realm) -> QuerySet:
     return occupied_streams
 
 
-def get_web_public_streams(realm: Realm) -> List[Dict[str, Any]]:  # nocoverage
+def get_web_public_streams(realm: Realm) -> List[APIStreamDict]:  # nocoverage
```

Comment:
@amanagr do you remember why this function has no test coverage?

Reply:
Yes, it is written in `lib/events.py`:

```
            else:
                # TODO: This line isn't used by the web app because it
                # gets these data via the `subscriptions` key; it will
                # be used when the mobile apps support logged-out
                # access.
                state["streams"] = get_web_public_streams(realm)  # nocoverage
```